packages/runtime/src/slate/BlockPlugin/__tests__/clearBlockKeyForDevice.test.tsx

```diff
@@ -1,6 +1,6 @@
+/** @jsxRuntime classic */
```

Comment:
Adding this comment prevents us from needing to use a tsconfig. Since we are already specifying the jsx pragma I like this approach in place of tsconfig.